packages/ui/src/FilesTable/index.tsx

```diff
@@ -11,6 +11,8 @@ type Messages = {
   deleteAction?: string;
   filenameColumnHeader?: string;
   descriptionColumnHeader?: string;
+  downloadCountHeader?: string;
+  lastDownloadedAtHeader?: string;
```

Comment:
Consistency in naming
```suggestion
  downloadCountColumnHeader?: string;
  lastDownloadedAtColumnHeader?: string;
```